src/core/Button/style.ts

```diff
@@ -30,13 +45,15 @@ const ButtonBase = styled(Button, {
 
     const padding = variant === "outlined" ? outlinedPadding : containedPadding;
 
+    const color = colors && colors[colorProp];
+
     return `
       padding: ${padding};
       min-width: 120px;
       height: 34px;
       &:hover, &:focus {
         color: white;
-        background-color: ${colors?.primary[500]};
+        background-color: ${color[500]};
```

Comment:
once we upgrade to MUI v5, which supports different `color` value other than `primary` and `secondary`, we can remove the custom css in SDS Button that v5 now does for us, and the change will be transparent to SDS library user, because the API is the same 🙆‍♂️